haskell-language-server.cabal

```diff
@@ -1635,148 +1675,151 @@ library hls-refactor-plugin
     PatternSynonyms
     RecordWildCards
     ViewPatterns
-  hs-source-dirs:   plugins/hls-refactor-plugin/src
-  build-depends:
-    , ghc
-    , bytestring
-    , ghc-boot
-    , regex-tdfa
-    , ghcide                == 2.11.0.0
-    , hls-plugin-api        == 2.11.0.0
-    , lsp
-    , text
-    , text-rope
-    , transformers
-    , unordered-containers
-    , containers
-    , ghc-exactprint < 1 || >= 1.4
-    , extra
-    , syb
-    , hls-graph
-    , dlist
-    , deepseq
-    , mtl
-    , lens
-    , time
+
+  hs-source-dirs: plugins/hls-refactor-plugin/src
+  build-depends:
     -- FIXME: Only needed to workaround for qualified imports in GHC 9.4
```

Comment:
This comment got floated up to the top of the `build-depends`, so it doesn't really make sense anymore. I would suggest something like this:

```
build-depends:
  bytestring,
  ...

-- FIXME: Only needed to workaround for qualified imports in GHC 9.4
build-depends:
  parser-combinators,
  regex-applicative
```